http/src/main/scala/org/apache/pekko/http/javadsl/server/directives/BasicDirectives.scala

```diff
@@ -170,7 +174,7 @@ abstract class BasicDirectives {
   def recoverRejectionsWith(
       f: JFunction[JIterable[Rejection], CompletionStage[RouteResult]], inner: Supplier[Route]): Route = RouteAdapter {
     D.recoverRejectionsWith(rs =>
-      f.apply(Util.javaArrayList(rs.map(_.asJava))).toScala.fast.map(_.asScala)(
+      FutureConverters.asScala(f.apply(Util.javaArrayList(rs.map(_.asJava)))).fast.map(_.asScala)(
```

Comment:
This was changed from using `.asScala` to `FutureConverters.asScala(stage)` because the `asScala` method from `pekko.util.FutureConverters._` was conflicting with an `org.apache.pekko.http.impl.util.AsScala.asScala` implicit.